### DISCUSSION TITLE
FOV Positioning for Gradient Modulated Adiabatic pulses

### DISCUSSION THREAD

**Discussion by wtclarke** [pulseq/pulseq#51] [General]
Dear all,

Is there any functionality in pulseq (and the Siemens interpreter) for positioning slice selective gradient modulated adiabatic pulses online on the scanner? These pulses require an additional non-linear phase to position at an arbitrary off-isocentre position, specifically the cumulative integral of the gradient as specified in eq[6] of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3214007/. I presume there isn't, but just wanted to check there wasn't any thing implemented, or something being worked on.

Best wishes,

Will

**Comment by m-a-x-i-m-z** [comment]
Dear Will, 

yes, it should be supported. Pulseq uses the principle, which is similar to the universal FOV positioning method of Magland and Wehrli (doi:10.1002/mrm.20933). We rely on Siemens' function "make FrequencyProportionalToGradient(), which I believe does exactly what you need, also for gradients which are time-dependent. There are some minor limitations related to the rate of change of the gradient because of the delays on the Siemens side, but if you are not using the highest slew rates during your RF pulse, then you should be on the safe side. 

Best , 
Maxim

**Reply by wtclarke** [reply]
Hi Maxim,

Ah great! I should have checked first, it just seemed like a niche use, but it sounds like some useful functionality in the IDEA code.

Thanks for the quick reply and sorry for my delay.
